portal/src/texts/core/pattern.md

```diff
@@ -1,21 +1,56 @@
+Vanlige handlinger er handlinger som gjentar seg uavhengig av komponenter og arbeidsflyter. Handlinger kan være tilbakevirkende (regressive), for eksempel Avbryt eller Slett, eller fremoverrettet (progressive), som Legg til eller Fortsett.
+Vi ønsker at designere og utviklere på tvers av Fremtind skal finne ferdige mønstre for slike handlinger. Da slipper dere å finne opp kruttet på nytt hver gang dere trenger vanlige handlinger i Jøkul til løsningene dere lager.
 
 ### Deaktiverte tilstander
 
 Vi lager mønstre for deaktiverte tilstander, slik at designere og utviklere vet nøyaktig hvordan det skal se ut når vi tar bort det interaktive momentet i en komponent.
 
 ### Filtrering
 
-Måten vi bruker, fjerner eller legger til data i et datasett, kan vi også standardisere til et mønster. Brukerne skal gjenkjenne hvordan de kan filtrere informasjon, uansett hvor i Fremtinds løsninger de er.
+Måten vi bruker, fjerner eller legger til data i et datasett på, kan vi også standardisere til et mønster. Brukerne skal gjenkjenne hvordan de kan filtrere informasjon, uansett hvor i Fremtinds løsninger de er.
 
 ### Overflytsregler
 
 Når en tekst er for lang for et bestemt område, kan vi bestemme hvordan vi vil at dette skal vises. Vi kan for eksempel forkorte informasjonen og vise en ellipse. I Jøkul finner du blant annet mønstre for hvordan vi forkorter søkebaner og lange URL-er.
+
+
+# Skjemadesign
+
+Skjemaer er viktige i Fremtinds løsninger og vi ønsker at de skal være konsekvente. Her finner du noen regler for  skjemaoppsett og hvordan vi bruker skjemakomponentene.
+
+## Skjemaseksjoner
+
+I Jøkul har vi flere skjemakomponenter, for eksempel
+- tekstfelt
+- radioknapper
+- avmerkingsbokser
+- nedtrekksfelt
+- datovelgere
+
+En eller flere skjemakomponenter sammen utgjør en _skjemaseksjon_, og et skjema består av en eller flere seksjoner. 
+
+Hver skjemakomponent har en _ledetekst_. Det kan være en overskrift med et spørsmål eller et ord eller en setning som sier noe om hva vi ønsker at brukeren skal oppgi eller gjøre. Deretter er det selve komponenten som brukeren skal gjøre noe med, og eventuelt en hjelpetekst eller melding. Ledetekstene kommer i to størrelser, stor og normal. Vi bruker den store varianten når skjemakomponenten er alene i en seksjon, og den normale når skjemakomponenten står sammen med flere komponenter i samme seksjon.
+
+Vi avgrenser seksjoner tydelig fra hverandre. Til det bruker vi enten luft, hvis seksjonene ligger mot en en felles bakgrunn, eller vi rammer dem inn i  egne kort. Hvis det er nødvendig, kan vi ha en overskrift på hver seksjon.
+
+De interne løsningene vi lager, kan ha mer kompakte skjemakomponenter. Det kan være nyttig hvis det er viktig å komprimere informasjonen vi gir i et skjermbilde, men hovedregelen er at vi også her skal følge prinsippene for skjemaoppsett.
```

Comment:
> komprimere informasjonen vi gir i et skjermbilde

Change this to the below text (or similar)? We should never compress content just to compress it 😊

> vise mye informasjon i et skjermbilde